app/scripts/models/notifications.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * The notifier broadcasts messages across multiple channels (iframe, tabs, browsers, etc).
+ */
+
+'use strict';
+
+define([
+  'backbone',
+  'underscore'
+], function (Backbone, _) {
+
+  var EVENTS = {
+    PROFILE_IMAGE_CHANGE: 'profile:image:change'
+  };
+
+  var Notifications = Backbone.Model.extend({
+    defaults: {
+    },
+    initialize: function (options) {
+      options = options || {};
+      this._channels = [
+        options.tabChannel,
+        options.webChannel
+      ];
+      // The iframe channel only available when the content is framed, naturally.
+      if (options.iframeChannel) {
+        this._channels.push(options.iframeChannel);
+      }
+
+      this._tabChannel = options.tabChannel;
```

Comment:
The only place the `this._tabChannel` reference is used is in `_listen`. `options.channel` could be passed in to `_listen` as an argument.